app/controllers/items/new.js

```diff
@@ -107,6 +120,36 @@ export default Ember.Controller.extend({
     return selected;
   }),
 
+  isInvalidaLabelCount: Ember.computed("labels", function() {
+    const labelCount = this.get("labels");
+    return !labelCount || Number(labelCount) < 0;
+  }),
+
+  isInvalidPrintCount: Ember.computed("labels", function() {
+    const labelCount = Number(this.get("labels"));
+    return _.inRange(labelCount, 0, 301);
+  }),
+
+  isMultipleCountPrint: Ember.computed("labels", function() {
+    const labelCount = Number(this.get("labels"));
+    return _.inRange(labelCount, 1, 301);
+  }),
+
+  isInvalidDimension: Ember.computed("length", "width", "height", function() {
+    const length = this.get("length");
+    const width = this.get("width");
+    const height = this.get("height");
+    const dimensionsCount = _.filter(
+      [length, width, height],
+      item => Number(item) <= 0
+    ).length;
+    return _.inRange(dimensionsCount, 1, 3);
+  }),
+
+  printLabelCount: Ember.computed("labels", function() {
+    return (this.get("labels") * 1).toString();
```

Comment:
This should be sufficient
```suggestion
    return this.get("labels").toString();
```

Reply:
This change was for following reason.

We wanted to remove the initial 0's from labels value and display.

For example
```
this.get('labels') gives `"00999"`

So we want to display `"999"` only

Hence `("00999" * 1).toString()` gives `"999"`
```

Reply:
Maybe add a comment, there's a change someone might remove it otherwise 😄 

Other potential options:
- `Number(this.get('labels'))`
- `this.get('labels').replace(/^0+/, '')`

Reply:
`Number(this.get('labels'))` looks much more readable and clean. 👍